server/database/PgRepositories.js

```diff
@@ -0,0 +1,62 @@
+const BaseRepository = require('./BaseRepository');
+
+class RawCaptureRepository extends BaseRepository {
+  constructor(session) {
+    super('raw_capture', session);
+    this._tableName = 'raw_capture';
+    this._session = session;
+  }
+
+  async getByFilter(filterCriteria, options) {
+    console.log('PG REPOSITORY DB getByFilter', filterCriteria, options);
+    const query = !!Object.keys(filterCriteria).length
+      ? filterCriteria
+      : `id` > 10;
+    return await this._session
+      .getDB()
+      .where(filterCriteria)
+      .select(
+        'id',
```

Comment:
The column names should match what exists as columns in schema `treetracker.capture`